src/main/java/com/trueffect/logica/PersonLogic.java

```diff
@@ -0,0 +1,116 @@
+package com.trueffect.logica;
+
+import com.trueffect.conection.db.DataBasePostgres;
+import com.trueffect.conection.db.OperationDataBase;
+import com.trueffect.messages.Message;
+import com.trueffect.model.Job;
+import com.trueffect.model.Person;
+import com.trueffect.response.ErrorResponse;
+import com.trueffect.sql.crud.JobCrud;
+import com.trueffect.sql.crud.PersonCrud;
+import com.trueffect.tools.CodeStatus;
+import com.trueffect.tools.DataResourse;
+import com.trueffect.tools.DataResourse.EmployeeWithPermissionModify;
+import com.trueffect.util.DataCondition;
+import com.trueffect.util.ErrorContainer;
+import com.trueffect.validation.RenterUserUpdate;
+import java.sql.Connection;
+
+/**
+ * @author santiago.mamani
+ */
+public class PersonLogic {
+
+    public Person createPerson(int id, Person person, DataCondition conditiondata) throws Exception {
+        Person personRes = null;
+        ErrorContainer errorContainer = new ErrorContainer();
+        //open conection 
+        Connection connection = DataBasePostgres.getConection();
+        try {
+            //Validation of data 
+            conditiondata.complyCondition(person);
+            PersonValidationsDB.veriryDataInDataBase(connection, person);
+            personRes = PersonCrud.insertrenterUser(connection, id, person);
```

Comment:
Please remove this : Person personRes=null changed to :
Person  personRes = PersonCrud.insertrenterUser(connection, id, person);